cerbero/utils/shell.py

```diff
@@ -205,12 +206,15 @@ def download(url, destination=None, recursive=False, check_cert=True, user=None,
       url = mirror_url
     cmd = "wget %s " % url
     path = None
+    dst = destination
     if recursive:
         cmd += "-r "
-        path = destination
+        path = dst
     else:
-        if destination is not None:
-            cmd += "-O %s " % destination
+        if not dst:
+            dst = ntpath.basename(url)
+        tmp_dst = "%s_tmp" % dst
+        cmd += "-O %s " % tmp_dst
```

Comment:
wont it be better to have one name for the temp file to avoid having cancelled downloads remaining and in a common path ?

Reply:
Thought about it but I think it's no necessary. Temporary file will be overwritten during next download try.